osu.Game/Screens/SelectV2/BeatmapCarousel.cs

```diff
@@ -67,8 +70,10 @@ protected override float GetSpacingBetweenPanels(CarouselItem top, CarouselItem
             return -SPACING;
         }
 
-        public BeatmapCarousel()
+        public BeatmapCarousel(bool waitForInitialCriteria = false)
```

Comment:
Why is this a constructor parameter?

Reply:
init property instead?

Reply:
My question is more of a "why does this need to be adjustable at all".